src/app/(main)/dashboard/components/Calendar/Calendar.logic.ts

```diff
@@ -159,6 +162,30 @@ export const useCalendarLogic = (
     }
   }
 
+  const weeklyCheckInLink = () => {
+    if (currentMeetingId && teamId && currentSprintNumber) {
+      return routePaths.weeklyCheckInPage(
+        teamId,
+        currentSprintNumber?.toString(),
+        currentMeetingId?.toString(),
```

Comment:
Is there a reason you added currentMeetingId here? It doesn't seem necessary and also the route paths is only expecting 2 arguments but you provided 3. Again, I tested with omitting the meetingid and it works so it'd prob be best to just remove that from here

Reply:
@timothyrusso 😅 in my recent pr I removed `meetingId` from `routePaths.weeklyCheckInPage` and `routePaths.submitVoyagePage`, because users can actually submit `weekly checkin` and `submit voyage` forms without creating a meeting

Reply:
ah ok, that's what it was